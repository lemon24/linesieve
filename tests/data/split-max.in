-s '(\S+):$'
split -s one -n1 -D:
split -s fixed -n1 -F -d'\s' -D:
split -s pattern -n2 -d'\W+' -D:
---
one:
a b c
fixed:
a\sb\sc
pattern:
a-b#c*d
