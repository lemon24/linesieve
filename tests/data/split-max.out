one
a:b c
fixed
a:b\sc
pattern
a:b:c*d
